setup.cfg

```diff
@@ -0,0 +1,82 @@
+[flake8]
+ignore =
+    # whitespace before ':'
+    E203,
+    # too many leading ### in a block comment
+    E501,
+    # Line break occurred before a binary operator (this is not PEP8 compatible)
+    W503,
+    # do not enforce existence of docstrings
+    D100,
+    D101,
+    D102,
+    D103,
+    D104,
+    D105,
+    D106,
+    D107,
+    # needed because of https://github.com/ambv/black/issues/144
+    D202,
+    # other string does contain unindexed parameters
+    P103
+max-line-length = 88
+exclude = migrations snapshots
+max-complexity = 10
+doctests = True
+
+[tool:isort]
+skip=migrations,snapshots
+known_first_party=caluma
+multi_line_output=3
+include_trailing_comma=True
+force_grid_wrap=0
+combine_as_imports=True
+line_length=88
+
+[tool:pytest]
+addopts = --reuse-db --randomly-seed=1521188766 --randomly-dont-reorganize
+DJANGO_SETTINGS_MODULE = caluma.settings
+env =
+    META_FIELDS=test-key,foobar
+    OIDC_USERINFO_ENDPOINT=mock://caluma.io/openid/userinfo
+    OIDC_INTROSPECT_ENDPOINT=mock://caluma.io/openid/introspect
+    OIDC_BEARER_TOKEN_REVALIDATION_TIME=60
+    LANGUAGES=en,de,fr
+    SIMPLE_HISTORY_ACTIVE=true
+filterwarnings =
+    error::DeprecationWarning
+    error::PendingDeprecationWarning
+
+# This is here because https://github.com/palantir/python-language-server is not
+# flexible enough to support flake8 and pycodestyle/pydocstyle at the same time.
+[pycodestyle]
+ignore =
+    # whitespace before ':'
```

Comment:
why do we need `pycodestyle` and `flake8` settings? A bit of duplication.

Reply:
* flake8 and pycodestyle/pydocstyle sections are partly redudant, because it is
  difficult to switch the palantir-python-language-server between flake8
  and py*style, in addition flake8 isn't correctly supported in the
  language-server.

-> I spent another hour trying to get flake8+python-language-server working -> no chance. I'd be very glad if I could have the configs in the project, otherwise I have to manually change setup.cfg and remember not to check it in etc... but I can understand if we don't want that, because the CI looks at [flake8]

Reply:
@sliverc I removed the py*style sections :cry:

After CI is done it should be ready to merge.